### PR TITLE
Renames the Bible to Sacred Tome + edits icon to be religiously neutral

### PR DESCRIPTION
**Pull request by cheekybrdy**
## About the PR 
[INPUT WANTED] [CONTROVERSIAL]
Uses the sprite made by what7431 from the Chaplain Additions project and renames bibles to the sacred texts, both internally and  externally.
dewestern

## Why's this needed? 
Puts the bible in a more religiously neutral position in regards to irl religions.

## Changelog 

```changelog
(u)Cheekybrdy
(*)Bibles are now called the Sacred Texts and have a religiously neutral icon. Sprite by what 7431.
```

### CODE REVIEW

File: code/obj/item/storage/bible.dm [hunk 1]
```diff
@@ -39,9 +39,9 @@
 			if (1 to 8)
 				target.visible_message(SPAN_ALERT("<B>[user] heals [target] mending [his_or_her(target)] wounds!</B>"))
 			if (9 to 15)
-				target.visible_message(SPAN_ALERT("<B>[user] heals [target] with the power of Christ!</B>"))
+				target.visible_message(SPAN_ALERT("<B>[user] heals [target] with the power of the Spirits!</B>"))
 			if (16 to 24)
-				target.visible_message(SPAN_ALERT("<B>[user] heals [target] by the will of the LORD!</B>"))
+				target.visible_message(SPAN_ALERT("<B>[user] heals [target] by the will of the the HOLY ONE!</B>"))
```
Comment (JORJ949) [inline code/obj/item/storage/bible.dm:44]: the the holy one

File: code/datums/hud/storage.dm [hunk 2]
```diff
@@ -156,7 +156,7 @@
 			sx = num_contents + 1
 			sy = 1
 
-			if (turfd) // goddamn BIBLES (prevents conflicting positions within different bibles)
+			if (turfd) // goddamn sacred_textsS (prevents conflicting positions within different sacred_textss)
```
Comment (Valtsu0) [inline code/datums/hud/storage.dm:159]: texts is already plural

**Comment by Valtsu0** [comment]
Atheist trait icon also has a cross

**Comment by cheekybrdy** [comment]
Forwarding from forums:
I'm seeing a bit of retreaded ground here so I'll pull up some context on what got the alternate books changed from being bibles with a dev quote.

"I think I said this in the Discord already but I'll say again here: I don't personally (this is my opinion and not an Official Dev View) like the idea of adding lots of reskins to Bibles, since Bibles have tons and tons of unique mechanics tied to them. They kill you instantly if you fart on them, have shared storage, can contain the chaplain's gun, "heal" people, and can damage two specific antags (one of which can't even hold the thing without giving themselves away and taking damage). I think an item with this many important features tied to it should have very consistent visuals, and I don't think making it less clear which item is/isn't a holy book that does all these things is worth the RP potential, especially when you can already make custom books."

Seeing contradictory dev quotes here now so pinch of salt for current dev opinions, but this is just meant to port the bible neutralisation to the "gameplay book" and leave the rest as flavour for the reasons above. I can see the current name I've proposed being a bit touch on the nose though, I'd say "Holy Texts" but I think that might be /tg's current name for it.

**Comment by cheekybrdy** [comment]
!merge_upstream

**Comment by cheekybrdy** [comment]
Looks like Taoism literally has sacred texts... time to re-rename

**Comment by cheekybrdy** [comment]
!merge_upstream

**Comment by cheekybrdy** [comment]
how the fuck did secret change waaaaa

**Comment by cheekybrdy** [comment]
closing cause idfk how I have a merge conflict with secret and to add scriptures

**Pull request closed without merging**